tests/test_ilastik/helpers/shellGuiTestCaseBase.py

```diff
@@ -130,15 +131,38 @@ def waitForViews(self, views):
         # Let the GUI catch up: Process all events
         QApplication.processEvents()
 
+    @contextlib.contextmanager
+    def hiddenCursor(self, imgView: ImageView2D):
+        """Context manager that hides the given image view's cursor
+
+        Useful, e.g. when accessing the viewports rendered image - where
+        sampling the cursor would be unwanted.
+        """
+        try:
+            imgView._crossHairCursor.setVisible(False)
+            # Wait for the gui to catch up
+            QApplication.processEvents()
+            self.waitForViews([imgView])
+
+            yield
+        finally:
+            imgView._crossHairCursor.setVisible(True)
+            # Wait for the gui to catch up
+            QApplication.processEvents()
+            self.waitForViews([imgView])
```

Comment:
If the cursor could be already hidden for some reason:
```suggestion
        wasVisible = imgView._crossHairCursor.isVisible()
        try:
            imgView._crossHairCursor.setVisible(False)
            # Wait for the gui to catch up
            QApplication.processEvents()
            self.waitForViews([imgView])
            yield
        finally:
            imgView._crossHairCursor.setVisible(wasVisible)
            # Wait for the gui to catch up
            QApplication.processEvents()
            self.waitForViews([imgView])
```

Reply:
yeah true, that could make sense (e.g. when doing some nesting with the contextmanager...)